addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -61,7 +61,7 @@ Wysiwyg.include({
                 $('.o_edit_menu_popover').find('[data-toggle="tooltip"]').tooltip('hide');
             }
 
-            if ($target.is('a') && !$target.attr('data-oe-model') && !$target.find('> [data-oe-model]').length && $target.closest('#wrapwrap').length) {
+            if ($target.is('a:not(.dropdown-toggle)') && !$target.attr('data-oe-model') && !$target.find('> [data-oe-model]').length && $target.closest('#wrapwrap').length) {
```

Comment:
Shouldn't we add the `o_no_link_popover` class on the user menu ?
If you want to make it generic to every dropdown-toggle, add that class on `noLinkPopoverClass` in link_popover_widget.